### ISSUE TITLE
Allow boosting of hotwater

### ISSUE THREAD

**Issue by asantaga** [asantaga/wiserHomeAssistantPlatform#82]
Create new service which allows the boosting of hotwater in a similar way to boosting heating ( service `wiser.boost_heating`)

@TobyLL fyi

**Comment by msp1974** [comment]
This is in some ways related to issue #80 and probably need to decide way forward on that before tackling this one.

**Comment by J1mbo** [comment]
Hi, +1 on this - though I suppose we can use an automation to return it to auto after an hour in the meantime.

When creating an automation I get "Message malformed: expected dict for dictionary value @ data['action'][0]['data']" - what is the syntax for the "data" field on the wiser.set_hotwater_mode service please?

**Comment by msp1974** [comment]
I think it should be 
hotwater_mode : auto|on|off

**Comment by J1mbo** [comment]
Perfect, thank you @msp1974

**Comment by asantaga** [comment]
This has been open a while, is it still an issue?

**Comment by msp1974** [comment]
Will work on adding this service next

**Comment by LewisSpring** [comment]
Hi there!
Any updates on this feature?

**Comment by rlewis187** [comment]
Here is my workaround in the form of a script:
```
alias: Boost Hot water for one hour
sequence:
  - service: wiser.set_hotwater_mode
    data:
      hotwater_mode: 'on'
  - delay:
      hours: 0
      minutes: 59
      seconds: 0
      milliseconds: 0
  - service: wiser.set_hotwater_mode
    data:
      hotwater_mode: auto
mode: single
icon: mdi:water
```

**Comment by LewisSpring** [comment]
Cheers!
I'll have to see if I can bodge that into a switch too.

**Comment by rlewis187** [comment]
This is what I use for calling the script in a lovelace button. 

Press and hold returns the hot water setting to auto:

```
type: button
tap_action:
  action: call-service
  service: script.boost_hot_water_for_one_hour
  service_data: {}
  target: {}
entity: sensor.wiser_hot_water
name: Boost Hot Water
show_state: false
hold_action:
  action: call-service
  service: wiser.set_hotwater_mode
  service_data:
    hotwater_mode: auto
  target: {}
```

**Comment by msp1974** [comment]
Just to say that i have been working on a significant upgrade to the api and this feature will be coming soon.

**Comment by asantaga** [comment]
fixed in 3.0
